Cogs/Other.py

```diff
@@ -264,26 +355,32 @@ async def HelpMe(self, ctx: commands.Context, command_type: typing.Optional[str]
                                   "message by message.",
                             inline=False)
         mod_embed.set_footer(
-            text="3/3 - Note: If you believe that there is an error with the bot, please let Jack or a moderator know "
-                 "in order to resolve it. Thank You!")
+            text="3/4")
         try:
             command_type = command_type.lower()
             if command_type == "all":
                 await ctx.author.send(embed=anyone_embed)
                 await ctx.author.send(embed=st_embed)
+                await ctx.author.send(embed=ts_embed)
                 await ctx.author.send(embed=mod_embed)
             elif command_type == "anyone":
                 await ctx.author.send(embed=anyone_embed)
             elif command_type == "st":
                 await ctx.author.send(embed=st_embed)
+            elif command_type == "townsquare":
+                await ctx.author.send(embed=ts_embed)
             elif command_type == "mod":
                 await ctx.author.send(embed=mod_embed)
             elif command_type == "no-mod":
                 await ctx.author.send(embed=anyone_embed)
                 await ctx.author.send(embed=st_embed)
+                await ctx.author.send(embed=ts_embed)
             else:
-                await ctx.author.send('Use `all`, `anyone`, `st`, `mod` or `no-mod` to filter the help message. '
-                                      'Default is `no-mod`.')
-
+                await ctx.author.send(
+                    'Use `all`, `anyone`, `st`, `townsquare`, `mod` or `no-mod` to filter the help message. '
+                    'Default is `no-mod`.')
+            await ctx.author.send("Note: If you believe that there is an error with the bot, please let Jack or a "
```

Comment:
I think given your involvement, & that we're now open source as a link to the issues page?